apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -158,6 +184,73 @@ const downloadButtonStyle = {
   textAlign: 'center'
 };
 
+class MicrobitDescription extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>The micro:bit</h2>
+        <center>
+          <a href={'https://microbit.org/'}>
```

Comment:
curly braces are only needed when the value of your HTML attribute contains JavaScript (i.e., isn't a hardcoded string). here and in other places, the value is a string, so this should be `<a href="https://microbit.org/">`

Reply:
Completed